app/src/main/java/com/example/vybe/MyVibesActivity.java

```diff
@@ -175,7 +178,7 @@ public void onNothingSelected(AdapterView<?> adapterView) {}
         myVibesAdapter = new MyVibesAdapter(this, R.layout.my_vibe_item, vibeEventList);
         vibesListView.setAdapter(myVibesAdapter);
 
-        final CollectionReference collectionReference = db.collection("VibeEvent");
+        final CollectionReference collectionReference = db.collection(vibeEventDBPath);
```

Comment:
Why is this declared final?

Reply:
Because lol. I removed it and everything works fines